cmd/clusterawsadm/cloudformation/bootstrap/iam.go

```diff
@@ -71,6 +71,18 @@ func (t Template) policyFunctionMap() map[PolicyName]func() *iamv1.PolicyDocumen
 	}
 }
 
+func (t Template) PrintPolicyDocs() error {
```

Comment:
I'm beginning to think this is some kind of attempt to undermine security because I still don't see anything changing here regardless of the force push.